### PR TITLE
elasticsearch.scroll-timeout example correction

### PR DESCRIPTION
**Pull request by farukarakaya**
originally mentioned [here](https://github.com/prestodb/prestodb.github.io/pull/42) 
@aweisberg

### CODE REVIEW

**Comment by zhenxiao** [comment]
thank you @farukarakaya 
looks good. could you please sign the CLA?

File: presto-docs/src/main/sphinx/connector/elasticsearch.rst [hunk 1]
```diff
@@ -30,7 +30,7 @@ replacing the properties as appropriate:
     elasticsearch.default-schema=default
     elasticsearch.table-description-directory=etc/elasticsearch/
     elasticsearch.scroll-size=1000
-    elasticsearch.scroll-timeout=60000
+    elasticsearch.scroll-timeout=60000ms
```
Comment (zhenxiao) [inline presto-docs/src/main/sphinx/connector/elasticsearch.rst]: could we set it a different value? 60000ms is 60s, which is 1 min, too long for scroll timeout
how about we set it a smaller number, e.g. 2s?

Comment (zhenxiao) [reply presto-docs/src/main/sphinx/connector/elasticsearch.rst]: how about we also delete the (ms) in line 49 and line 83? it is misleading
also, in 85, the default value is 1s, not 20s, could you please fix it?

Comment (farukarakaya) [reply presto-docs/src/main/sphinx/connector/elasticsearch.rst]: Sure, I did it. I didn't mean to discuss if values are appropriate but the notation is wrong and example is not working. Anyway, it's good to fix those too.

**Pull request merged**